lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -482,6 +482,97 @@ def operations
       end
     end # End of POLICY_ELEMENT_TYPES iteration
 
+    # Arel matches provides agnostic case insensitive sql for MySQL and
```

Comment:
I would move this comment to line 487, where we actually see arel in use. For a comment on the method definition itself, maybe something like "Given a policy element class and a set of conditions, returns an ActiveRecord::Relation with those conditions applied".